File: 2018/MarchLongChallenge/BIGSALE.py
#!/usr/bin/env python
t = int(input())
while(t>0):
	t-=1
	n = int(input())
	ans = 0.0
	for i in range(0,n):
		inp = input().split(' ')
		qty = int(inp[0])
		price = float(inp[1])
		x = float(inp[2])
		ans = ans + qty * price * (x * x / 10000.0)
	print(ans)

File: 2018/MarchLongChallenge/BIGSALE.cpp
#include <bits/stdc++.h>
using namespace std;

#define sz 100005

int main()
{
	int t;
	cin>>t;
	while(t--)
	{
		double ans = 0;
		int n;
		cin>>n;
		for(int i=0; i<n; i++)
		{
			double price, qty, x;
			cin>>price>>qty>>x;
			ans = ans + qty * price * x * x;
		}
		cout<<(ans/10000.0)<<"\n";
	}
	return 0;
}
